### PR TITLE
GA4 implementation record updates

### PR DESCRIPTION
**Pull request by andysellick**
## What
Update some of the information in the GA4 implementation record section, specifically:

- smart answers
- get emails and RSS link clicks

Also adjust how examples are contained in data and displayed, and adds a 'further reading' page in the docs listing our blog posts to date.

## Why
Part of the work to document our GA4 approach.

Trello card: https://trello.com/c/JbLX3n2C/388-implementation-record-additions-and-improvements

### CODE REVIEW

**Comment by andysellick** [comment]
@AshGDS @JamesCGDS sorry, hold off on reviewing this for a moment - just remembered a small feature I'd like to add.

**Comment by andysellick** [comment]
Okay, all good to review now, thanks 👍

File: data/analytics/navigation.yml [hunk 1]
```diff
@@ -41,6 +43,10 @@ trackers:
   name: Trackers
   desc: Trackers are distinct JavaScripts that collect GA4 data in specific ways. Each works differently but produces and sends data to GA4 in the same way.
 
+further_docs:
+  name: Further reading
+  desc: blah
```
Comment (AshGDS) [inline data/analytics/navigation.yml:48]: Is this meant to be here?

**Review by AshGDS — CHANGES_REQUESTED** [review]
Had a glance and looks OK, had a small comment though. Thanks for adding more docs!

File: data/analytics/navigation.yml (hunk 1 above)
Comment (andysellick) [reply data/analytics/navigation.yml]: Whoops! Sorry, no it isn't. Removed.

**Review by AshGDS — APPROVED** [review]
LGTM :+1:

**Pull request merged**